package.json

```diff
@@ -43,22 +43,22 @@
     "sdk-build:win32": "echo 'NO WIN32 BUILD'",
     "sdk-build:default": "rimraf ./dist/sdk-core && cd ./libs/sdk-core && rollup -c && cp package.json README.md ../../dist/sdk-core && cd - && cp ./LICENSE.md dist/sdk-core",
     "widget:build": "ng build widget --prod --single-bundle --output-hashing none",
-    "postinstall": "ngcc --properties es2015 browser module main"
+    "postinstall": "ngcc --properties es2020 browser module main"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "13.3.11",
-    "@angular/cdk": "13.3.9",
-    "@angular/common": "13.3.11",
-    "@angular/compiler": "13.3.11",
-    "@angular/core": "13.3.11",
+    "@angular/animations": "14.0.3",
+    "@angular/cdk": "14.0.3",
+    "@angular/common": "14.0.3",
+    "@angular/compiler": "14.0.3",
+    "@angular/core": "14.0.3",
     "@angular/flex-layout": "^13.0.0-beta.37",
```

Comment:
you should migrate all `@angular` deps: `@angular/flex-layout`, `@angular/youtube-player`, `@angular-eslint`,…